server/src/main/java/com/exacaster/lighter/configuration/AppConfiguration.java

```diff
@@ -34,6 +34,7 @@ public class AppConfiguration {
     private final Integer pyGatewayPort;
     @JsonProperty(access = Access.WRITE_ONLY)
     private final String url;
+    private final Duration lighterZombieInterval;
```

Comment:
I dint think “lighter” prefix is needed for property name, it is specified as a prefix on the annotation.

Reply:
thanks